modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1216,8 +1301,8 @@ Topology Display Filters Extension
 [cols=",,,",options="header",]
 |===
 |Name |Value Type |Optional |Description
-|`getTopologyFilters` |`CodeRef<() => TopologyDisplayOption[]>` |no |
-|`applyDisplayOptions` |`CodeRef<TopologyApplyDisplayOptions>` |no |
+|`getTopologyFilters` |`CodeRef<() => TopologyDisplayOption[]>` |no | Getter for topology filters specific to the extension |
+|`applyDisplayOptions` |`CodeRef<TopologyApplyDisplayOptions>` |no | Function to apply filters to the model |
```

Comment:
formatting issue in table
![Screenshot 2023-03-30 at 2 16 29 PM](https://user-images.githubusercontent.com/12692381/228746369-76c240b0-57cc-4e6c-9ec3-cedc25c720ad.png)